src/Microsoft.Extensions.Logging/ILoggerBuilder.cs

```diff
@@ -0,0 +1,22 @@
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Extensions.Logging
+{
+    public interface ILoggerBuilder
```

Comment:
We want to prevent all logging related methods appearing on `IServiceCollection`. If we keep them on `IServiceCollection` they would also need to be named in more verbose way (AddConsoleLogger, AddLoggingFilter, etc. )